### ISSUE TITLE
Class based routes mishandles parameter in name (edge case)

### ISSUE THREAD

**Issue by ordago** [laravel-shift/workbench-issues#45]
**Desktop:**
 - OS: Windows
 - OS Version: 10
 - App Version: 0.6.0.0

**Describe the bug**
While upgrading an app with a non conventional route naming convention I found the the names were not converted correctly.

**To Reproduce**
3. Run Class based routes task

```diff
-Route::get('/event/{event}', 'EventController@show')->name('events@show');
+Route::get('/event/{event}', [EventController::class, 'show'])->name([events::class, 'show']);
```

![image](https://user-images.githubusercontent.com/6376814/135443765-7824467f-88c4-4717-93a3-4f9577edda5e.png)

**Comment by jasonmccreary** [comment]
Interesting. I didn't even know you could name routes with that syntax. 🤔

**Comment by ordago** [comment]
![image](https://user-images.githubusercontent.com/6376814/135942160-a07a643c-bb59-40e1-8380-57a267e80d97.png)

It seems all sort of unadvisable things can be done 😅

**Comment by jasonmccreary** [comment]
This should be patched in the latest release (0.6.1). Let us know if not.
